### PR TITLE
Adds GloryHoleSwallows xPath scraper

### PR DESCRIPTION
**Pull request by halorrr**
I'd like some input on this one. I'm unclear why the date/image does not work in testing.

### CODE REVIEW

File: scrapers/GloryHoleSwallow.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          - regex: .+(?:<.+>)(.+)(?:<.+>)
+          - with: $1
+        parseDate: January 02, 2006
+      Tags:
+        Name: //span[@class="tour_update_tags"]/a//font/text()
+      Image:
+        selector: (//div[@class="back"]/center/img/@src)[1]
+        replace:
+          - regex: ^
+          - with: https://gloryholeswallow.com/newtour/
```
Comment (bnkai) [inline scrapers/GloryHoleSwallow.yml:23]: you put an extra `-` to the `with:`
should be
`with: https://gloryholeswallow.com/newtour/`

**Comment by bnkai** [comment]
For the date part i think it's a bug in the scraper code ( comment nodes don't have innertext so output html should be used instead ) that i'll open an issue for.
For the image part the `-` was the  issue

File: scrapers/GloryHoleSwallow.yml (hunk 1 above)
Comment (halorrr) [reply scrapers/GloryHoleSwallow.yml]: Ahhh there we go

**Comment by halorrr** [comment]
Perfect, so only thing that doesn't work on this one is the date then (which is kinda a bonus feature anyway since the date isn't actually displayed visually on the pages, so it would be a welcome surprise that this scraper is actually able to pull a date once comments are fixed haha)

File: scrapers/GloryHoleSwallow.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Details: //span[@class="latest_update_description"]/text()
+      Date:
+        selector: //ul/comment()[contains(.,'update_date')]
+        replace:
+          - regex: .+(?:<.+>)(.+)(?:<.+>)
+            with: $1
+        parseDate: January 02, 2006
```
Comment (bnkai) [inline scrapers/GloryHoleSwallow.yml:16]: Dates used are in the 3 letter form
`January` should be switched to `Jan`

Comment (halorrr) [reply scrapers/GloryHoleSwallow.yml]: Done!

**Comment by halorrr** [comment]
Updated to include documentation as well

**Comment by bnkai** [comment]
Changes needed to stash were merged. Merging this.

**Pull request merged**